<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pause">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>暫停</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resume">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>繼續</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>結束</string>
    </property>
   </widget>
   <widget class="QLabel" name="smkrTobacco">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pic/Tobacco.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="smkrPaper">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pic/Paper.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="smkrMatch">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>450</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pic/Match.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Tobacco">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pic/T.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Paper">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>150</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pic/P.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Match">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>260</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pic/M.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Table">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>390</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pic/Table.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Agent">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>40</y>
      <width>430</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pic/Agent.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Tsurprise">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pic/surprise.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Psurprise">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pic/surprise.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Msurprise">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pic/surprise.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHi">
    <property name="title">
     <string>Hi</string>
    </property>
   </widget>
   <addaction name="menuHi"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
